test/unstoppable/Unstoppable_Halmos.t.sol

```diff
@@ -9,10 +9,9 @@ import {UnstoppableMonitor} from "../../src/unstoppable/UnstoppableMonitor.sol";
 import "./AbstractAttacker.sol";
 import "forge-std/console.sol";
 
-contract UnstoppableChallenge is Test {
-    address deployer = makeAddr("deployer");
-    address player = makeAddr("player");
-    address monitor = makeAddr("monitor");
+contract UnstoppableChallengeHalmos is Test {
+    address deployer = address(0xde4107e4);
+    address player = address(0xa77ac3e4);
```

Comment:
I think this is key to avoid some of the weird stuff you have seen. With makeAddr, we treat deployer and player as symbolic so they _could_ be the addresses of the deployer, the monitor contract, the vault, etc... This is a bit counterintuitive, but it's an implicit assumption when working with concrete addresses. 

If we hardcode them to fixed addresses, we get more readable traces and we also make them all distinct.

Reply:
Is it a right behavior? I thought that symbolic data should come EXCLUSIVELY from "svm".
It is clear for me that users, that potentially may make migrations from foundry to halmos tests will meet the same counterintuitive problem. So, maybe makeAddr should generate concrete address, just like foundry?

Reply:
> I thought that symbolic data should come EXCLUSIVELY from "svm"

not really, svm is just a convenient helper. Depending on the test configuration, there can be plenty of sources of symbolic values (extcodehash, extcodesize, tx.origin, uninitialized storage variables, gas, blockhash, hash/signatures)

> So, maybe makeAddr should generate concrete address, just like foundry?

we can potentially look into this. The problem is that `makeAddr` calls keccak under the hood, so we would need to change how we handle keccak in general

Reply:
using the 2 hardcoded addresses like I did is a little hacky, but I don't think it compromises the scenario and it does provide the benefit of showing as recognizable addresses in traces

Reply:
> using the 2 hardcoded addresses like I did is a little hacky, but I don't think it compromises the scenario and it does provide the benefit of showing as recognizable addresses in traces

Yes, sure. My question was that such makeAddr behavior may create potential problems for users. Thanks